content/ngrx/do-not-put-everything-in-the-store.md

```diff
@@ -16,7 +16,10 @@ The NgRx core team has come up with a principle called **SHARI**, that can be us
 - Retrieved: State that needs to be retrieved with a side effect, e.g. an HTTP request
 - Impacted: State that is impacted by other components
 
+Don't over-engineer your state management! Most of the times information comes along via Ajax or WebSockets and therefore is handled on the server side. Always ask yourself when and why to put something in a client side store and keep alternatives in mind. For example use routes to reflect applied filters on a list or use a `BehaviorSubject` in a service if you need to save some simple data like settings. Mike Ryan gave a very good talk on this topic: [You might not need NgRx](https://youtu.be/omnwu_etHTY)
+
 # Resources
 
 - [Reducing the Boilerplate with NgRx](https://www.youtube.com/watch?v=t3jx0EC-Y3c) by Mike Ryan and Brandon Roberts
 - [Do we really need @ngrx/store](https://blog.strongbrew.io/do-we-really-need-redux/) by Brecht Billiet
+- [Simple State Management with RxJS’s scan operator](https://juristr.com/blog/2018/10/simple-state-management-with-scan/)
```

Comment:
Please add the author --> `by Juri Strumpflohner`

Reply:
done